README.md

```diff
@@ -8,7 +7,7 @@ In app config:
     ...
 
     'mobileDetect' => array(
-        'class' => 'ext.MobileDetect.MobileDetect'
+        'class' => 'application.vendors.candasm.yii1-mobile-detect-component.src.MobileDetectComponent'
```

Comment:
I think, that there should be class (for example: `iamsalnikov\MobileDetect\MobileDetect`)